client/blocks/reader-feed-item/index.tsx

```diff
@@ -96,37 +97,79 @@ export default function ReaderFeedItem( props: ReaderFeedItemProps ): JSX.Elemen
 			return;
 		}
 
-		onSubscribe( {
-			blog_id: blogId ?? undefined,
-			feed_id: feedId,
-			url: subscribeUrl,
-			onSuccess: () => {
-				dispatch(
-					successNotice(
-						translate( 'Success! You are now subscribed to %s.', { args: filteredDisplayUrl } ),
-						{ duration: 5000 }
-					)
-				);
-
-				recordSearchSiteSubscribed( { blog_id: blogId, url: subscribeUrl, source } );
-
-				if ( shouldTrackRecommendedSearch ) {
-					// reader: action: site_followed, railcar, ui_algo, ui_position, fetch_algo, fetch_position, fetch_lang, rec_blog_id, (incorrect: only railcar & action accepted)
-					// subscriptions: action: recommended_search_item_site_subscribed, railcar
-					recordTrainTracksInteract( {
-						railcarId: railcar.railcar,
-						action: 'recommended_search_item_site_subscribed',
-					} );
+		const noticeOptions: NoticeOptions = { duration: 5000 };
+		if ( feed?.subscription_id ) {
```

Comment:
With noted confusion above between having both `feed` and `site` queries and what their results are being used for here, do we need any `site` case?

Reply:
No. Feed subscriptions are same for wpcom sites and non-wpcom sites.